flang/lib/Parser/preprocessor.cpp

```diff
@@ -46,6 +49,110 @@ bool Definition::set_isDisabled(bool disable) {
   return was;
 }
 
+void Definition::Print(
+    llvm::raw_ostream &out, llvm::StringRef macroName) const {
+  if (isDisabled_) {
```

Comment:
Macros are only temporarily disabled during expansion; not relevant here.

Reply:
Done.